packages/test/snapshots/package.json

```diff
@@ -81,8 +81,7 @@
 		"@fluidframework/replay-driver": "workspace:~",
 		"@fluidframework/sequence": "workspace:~",
 		"@fluidframework/server-local-server": "^7.0.0",
-		"@fluidframework/test-utils": "workspace:~",
-		"mocha": "^10.8.2"
```

Comment:
I think keep? There's one non-test file in the package that imports from mocha. I don't remember enough about the snapshot tests to know if it matters whether this is a devDep or not, so I would lean towards keeping it if the PR is just about upgrading other devdeps.

Reply:
good catch. bad merge.